app/routes/events/view.js

```diff
@@ -9,7 +9,7 @@ export default Route.extend({
 
   model(params) {
     return this.store.findRecord('event', params.event_id, {
-      include: 'event-topic,event-sub-topic,event-type,event-copyright,tax,sponsors'
+      include: 'event-topic,event-sub-topic,event-type,event-copyright,tax,sponsors,speakers-call,microlocations,tracks,session-types'
```

Comment:
Please don't load the whole bulk of data here. All this will be loaded unnecessarily for all pages under `/events/<id>/*`. Load what is needed at that specific child route level.